Tests/SentryTests/Integrations/OutOfMemory/SentryOutOfMemoryScopeObserverTests.swift

```diff
@@ -0,0 +1,131 @@
+import XCTest
+
+class SentryOutOfMemoryScopeObserverTests: XCTestCase {
+    private class Fixture {
+        let breadcrumb: Breadcrumb
+        let options: Options
+        let fileManager: SentryFileManager
+        let currentDate = TestCurrentDateProvider()
+
+        init() {
+            breadcrumb = Breadcrumb()
```

Comment:
`l`: You could use https://github.com/getsentry/sentry-cocoa/blob/725723e10c8d2448514e4cd4cee4e71351922e9f/Tests/SentryTests/Protocol/TestData.swift#L17-L25

Reply:
Actually I can't really, because that one contains a data dictionary, which means I can't force cast the serialized crumb to `[String: String]`, which means I can't do `XCTAssertEqual(dict, breadcrumb)` 😅

Reply:
You could use the one from TestData and then modify just `.data`.

Reply:
No problem, done.